roles/ci_gen_kustomize/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directories exist
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    mode: "0755"
+  loop:
+    - "{{ cifmw_ci_gen_kustomize_src_dir }}"
+    - "{{ cifmw_ci_gen_kustomize_dest_dir }}"
+
+- name: List snippets
+  register: _snippets
+  ansible.builtin.find:
+    paths: "{{ cifmw_ci_gen_kustomize_src_dir }}"
+    recurse: false
+
+- name: Load various snippet files
+  register: _snippets_content
+  ansible.builtin.slurp:
+    path: "{{ file.path }}"
+  loop: "{{ _snippets.files }}"
```

Comment:
Should we sort them?

Reply:
ah, yeah, might be good indeed. Allowing then to get some ordering and, eventually, overrides.